test/libsolidity/smtCheckerTests/literalSuffixes/division_by_zero_function_two_arguments.sol

```diff
@@ -0,0 +1,15 @@
+function suffix2(uint m, uint e) pure suffix returns (uint) {
+    return m / e;
+}
+
+contract C {
+    function test() public pure returns (uint x) {
+        x = 1234567890 suffix2;
+    }
+}
+
+// ====
+// SMTEngine: all
+// ----
+// Warning 6031: (159-177): Internal error: Expression undefined for SMT solver.
```

Comment:
It would be good to avoid this.

Reply:
This is probably the disconnect between the function call and the definition. I.e. we have only one argument in the AST (`1234567890`) but the function requires two. When the SMTChecker encounters this it should call `RationalNumberType::fractionalDecomposition()` to split it and treat the result as actual arguments.

Doing this would probably let us actually get rid of the warning because it's all that's needed to support those suffixes.